uitools/examples/qml_quick/src/demos/CoordinateConversionDemo.qml

```diff
@@ -31,7 +31,7 @@ DemoPage {
             }
         }
     }
-
+//! [Set up Coordinate Conversion QML]
```

Comment:
❌  Same comment here as Callout - better to leave out the `mapViewContents: Component` and only show the MapView and its children